### PR TITLE
Create update-interactive.html

### PR DESCRIPTION
**Pull request by zhangx501**
>^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
> Remember to delete this note before submitting your pull request.
>
> For pull requests on 1.15 Features: set Milestone to 1.15 and Base Branch to dev-1.15
> 
> For pull requests on Chinese localization, set Base Branch to release-1.14
>
> For pull requests on Korean Localization: set Base Branch to dev-1.14-ko.\<latest team milestone>
>
> If you need Help on editing and submitting pull requests, visit:
> https://kubernetes.io/docs/contribute/start/#improve-existing-content.
>
> If you need Help on choosing which branch to use, visit:
> https://kubernetes.io/docs/contribute/start#choose-which-git-branch-to-use.
>^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
>

### CODE REVIEW

File: content/zh/docs/tutorials/kubernetes-basics/update/update-interactive.html [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+<link href="/docs/tutorials/kubernetes-basics/public/css/styles.css" rel="stylesheet">
+<link href="/docs/tutorials/kubernetes-basics/public/css/overrides.css" rel="stylesheet">
+<script src="https://katacoda.com/embed.js"></script>
+<div class="layout" id="top">
+
+    <main class="content katacoda-content">
+    
+        <div class="katacoda">
+            <div class="katacoda__alert">
+                To interact with the Terminal, please use the desktop/tablet version
```
Comment (xichengliudui) [inline content/zh/docs/tutorials/kubernetes-basics/update/update-interactive.html]: here.

**Comment by zhangx501** [comment]
/assign @haibinxie
Please make the file get approval.

File: content/zh/docs/tutorials/kubernetes-basics/update/update-interactive.html [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+            </div>
+            <div class="katacoda__box" id="inline-terminal-1"  data-katacoda-id="kubernetes-bootcamp/6" data-katacoda-color="326de6" data-katacoda-secondary="273d6d" data-katacoda-hideintro="false" data-katacoda-font="Roboto" data-katacoda-fontheader="Roboto Slab" data-katacoda-prompt="Kubernetes Bootcamp Terminal" style="height: 600px;">
+            </div>
+        </div>
+        <div class="row">
+            <div class="col-md-12">
+                <a class="btn btn-lg btn-success" href="/docs/tutorials/kubernetes-basics/" role="button">Back to Kubernetes Basics<span class="btn__next">›</span></a>
```
Comment (xichengliudui) [inline content/zh/docs/tutorials/kubernetes-basics/update/update-interactive.html]: Back to Kubernetes Basics ->回到 Kubernetes 的基础

**Comment by xichengliudui** [comment]
You forgot something

**Comment by xiangpengzhao** [comment]
@zhangx501 please update the PR as per @xichengliudui 's inline comments.
/assign @chenrui333 
for approval after the PR is ready to merge.

**Comment by zhangx501** [comment]
@chenrui333
Please give this PR appoval.

**Comment by markthink** [comment]
@zhangx501  预览：https://deploy-preview-15867--k8s-v1-14.netlify.com/zh/docs/tutorials/kubernetes-basics/update/update-interactive/

左侧菜单丢失了..

**Comment by chenrui333** [comment]
内容方面没啥问题，没有做文档检查。

**Pull request merged**